Google Sheets DS Plugin Docs (#255)
grafana/google-sheets-datasource: docs/sources/sample-dashboard/_index.md

---

> title: Creating a sample Dashboard using the Google Sheets data source plugin for Grafana
> menuTitle: Creating a sample Dashboard
> description: Creating a sample Dashboard using the Google Sheets data source plugin to visualize Google Spreadsheets data in Grafana.
> keywords:
>   - data source
>   - google sheets
>   - spreadsheets
>   - xls data
>   - xlsx data
>   - excel sheets
>   - excel data
>   - csv data
>   - visualize spreadsheets
> labels:
>   products:
>     - oss
>     - enterprise
>     - cloud
> weight: 400
> ---
> 
> # Creating a sample Dashboard

Reviewer: Task topics should avoid gerunds: https://grafana.com/docs/writers-toolkit/review/lint-prose/rules/#grafanagerunds

```suggestion
title: Create a sample dashboard using the Google Sheets data source plugin for Grafana
menuTitle: Create a sample dashboard
description: Create a sample Dashboard using the Google Sheets data source plugin to visualize Google Spreadsheets data in Grafana.
keywords:
  - data source
  - google sheets
  - spreadsheets
  - xls data
  - xlsx data
  - excel sheets
  - excel data
  - csv data
  - visualize spreadsheets
labels:
  products:
    - oss
    - enterprise
    - cloud
weight: 400
---

# Create a sample dashboard using the Google Sheets data source plugin for Grafana
```

We are going to create a sample Dashboard by using this publicly available [demo spreadsheet](https://docs.google.com/spreadsheets/d/1TZlZX67Y0s4CvRro_3pCYqRCKuXer81oFp_xcsjPpe8/edit?usp=sharing) that is suitable for visualization in graphs and in tables.

## Example

> 1. Copy the sheet-ID from the demo spreadsheet which should look similiar to `1TZlZX67Y0s4CvRro_3pCYqRCKuXer81oFp_xcsjPpe8` in your web browser.

Reviewer 2: ```suggestion
1. Copy the spreadsheet ID from the demo spreadsheet which should look similar to `1TZlZX67Y0s4CvRro_3pCYqRCKuXer81oFp_xcsjPpe8` in your web browser.
```

1. Paste in the Query editor
   ![Paste the Spreadsheet ID in the query editor](/media/docs/plugins/google-sheets-example-1.png)

1. After that you should be able to see a similar Time series visualization as Grafana automatically detect this data as Time series and uses the Time Series panel visualziation to display it.
   ![View Spreadsheet data in Time Series panel visualization](/media/docs/plugins/google-sheets-example-2.png)

> 1. You can also use other visualization panel options for e.g. Bar Gauge:

Reviewer 2: ```suggestion
1. You can also use other visualization panel options for e.g. Bar gauge:
```

>    ![View Spreadsheet data in Bar Gauge panel visualization ](/media/docs/plugins/google-sheets-example-3.png)

Reviewer: I think this should be rewritten as a task using the task template.

Remember to:
- Use the second person "you", not first person plural "we"
- Ensure the text for each step properly explains what needs to be done. Screenshots can be helpful but the most valuable alt text for that image would be the step explaining what was done.
- Use *bold* for UI elements and fields, use `code` for values: https://grafana.com/docs/writers-toolkit/write/style-guide/style-conventions/#bold

Author: https://grafana.com/docs/writers-toolkit/structure/topic-types/task/

## Play demo

> The Play demo dashboards provides a reference the above sample dashboard and also allows you to modify and customized it on the fly.

Reviewer: ```suggestion
The Play demo dashboards provides a reference dashboard and allows you to modify and create your own custom dashboards.
```

> {{< docs/play title="GitHub data source plugin demo" url="https://play.grafana.org/dashboards/f/bb613d16-7ee5-4cf4-89ac-60dd9405fdd7/demo-github" >}}

Reviewer: This looks to be copied from the GitHub page, and needs to be updated for this data source.

Author: fixed now
